inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp

```diff
@@ -643,6 +643,38 @@ void MKLDNNMVNNode::getSupportedDescriptors() {
     } else if (details::CaselessEq<std::string>()(epsMode, "outside_sqrt")) {
         epsMode_ = outsideSqrt;
     }
+
+    SizeVector inputShape = getParentEdgeAt(0)->getDims().ToSizeVector();
```

Comment:
Minor suggestion - add 626th line:
`const SizeVector inputShape = getParentEdgeAt(0)->getDims().ToSizeVector();`
And change 627th line to:
`const auto& numOfDims = inputShape.size();`

Reply:
done, thank you.